src/main/steps/edge-case/date-of-birth/content.test.ts

```diff
@@ -126,7 +107,7 @@ it('should have dateOfBirth label when language: en and  applyingWith: alone', (
   const commonContent1 = { language: 'en', userCase: { applyingWith: 'alone' } } as CommonContent;
```

Comment:
1) do we need applyingWith?

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';